src/display/editor/editor.js

```diff
@@ -206,6 +208,25 @@ class AnnotationEditor {
    * @param {Object} l10n
    */
   static initialize(l10n, _uiManager, options) {
+    AnnotationEditor._l10nResizer ||= new Map(
```

Comment:
I know the code was already doing it, but creating IDs ad run-time is discouraged, because it makes maintenance unnecessarily harder:
* Searching the code for the full Fluent message ID won't find anything.
* You still need to implement a solution in case only a few of these strings need to change, without forcing retranslation off all strings.

Can you create a map when each action is mapped to a full ID?

Reply:
> I know the code was already doing it, but creating IDs at run-time is discouraged,
> [...]
> Can you create a map where each action is explicitly mapped to a full ID?

Done; given the old code I just wasn't aware of this requirement.